### ISSUE TITLE
Swipe not working when there are multiple sliders on single page

### ISSUE THREAD

**Issue by bbujisic** [iosscripts/iosslider#257]
Hi Mark,

First off -- you made one **awesome** product. Thank you so much!

Seems like swiping feature does not work when there are two sliders in one page. My dev server: http://peoplematter.circle-outsourcing.info/

DesktopClickDrag thing works just fine, but touch swipe (tested in two Androids and in desktop chrome with touch emulation) does not.

The code I used to bootstrap iosSlider:

``` javascript
$('.iosContainer').each(function(){
    $(this).find('.iosSlider').iosSlider({
      snapToChildren: true,
      scrollbar: false,
      desktopClickDrag: true,
      responsiveSlides: true,
      responsiveSlideContainer: false,
      navNextSelector: $(this).find('.right'),
      navPrevSelector: $(this).find('.left')
    });
  })
```

Thanks!
Branislav

**Comment by marcwhitbread** [comment]
Hmmm. Cant seem to reproduce the issue in my benchmarks. Must be something specific to your implementation. Take a look at the benchmarks folder in the download package. There are tons of sliders there on the same page. 

Also, take a look at example 5, double slider gallery. There are 2 sliders on that page and they both work via touch.

**Comment by bbujisic** [comment]
Seems like I found a bug in Android's default browser. At least in version I have.

I tried your benchmarks and swiping still **did not work** on my Samsung Galaxy S III. Same cell phone but Chrome instead default browser -- thing worked in both my and your implementation!

Crazy stuff, but not your responsibility. It is interesting problem, though. I will try to play with it more if I find time.

Thanks for fast reply!
